pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -198,25 +209,34 @@ const (
 
 // BuildStatus is the status for a Build resource
 type BuildStatus struct {
+	// +optional
 	Builder BuildProvider `json:"builder,omitempty"`
 
 	// Cluster provides additional information if the builder is Cluster.
+	// +optional
 	Cluster *ClusterSpec `json:"cluster,omitempty"`
+
 	// Google provides additional information if the builder is Google.
+	// +optional
 	Google *GoogleSpec `json:"google,omitempty"`
 
 	// StartTime is the time the build is actually started.
+	// +optional
 	StartTime metav1.Time `json:"startTime,omitEmpty"`
```

Comment:
I think `StartTime` and `CompletionTime ` should be a pointer

see an example here:

https://github.com/kubernetes/kubernetes/blob/7f23a743e8c23ac6489340bbb34fa6f1d392db9d/staging/src/k8s.io/apiserver/pkg/apis/example/types.go#L70

Reply:
I have updated PR to change startTime and completeTime to be pointer